service/src/main/kotlin/io/provenance/explorer/domain/entities/Markers.kt

```diff
@@ -219,7 +219,7 @@ object AssetPricingTable : IdTable<Int>(name = "asset_pricing") {
     val pricing = decimal("pricing", 100, 50)
     val pricingDenom = varchar("pricing_denom", 256)
     val lastUpdated = datetime("last_updated")
-    val data = jsonb<AssetPricingTable, AssetPricing>("data", OBJECT_MAPPER)
+    val data = jsonb<AssetPricingTable, AssetPricing>("data", OBJECT_MAPPER).nullable()
```

Comment:
Just verifying since I don't see a DB migration here, the current column in the db is nullable?

Reply:
Yes, it is.  I just needed a way to set it on the kotlin side to null for updates.

I will be removing this column in the future anyway: https://github.com/provenance-io/explorer-service/issues/563

It was saving the json that was returned from figure pricing, which didn't have any extra information that wasn't already being saved as a column.